.taskcluster.yml

```diff
@@ -261,12 +261,13 @@ tasks:
                                         $if: 'tasks_for == "action"'
                                         then: >
                                             PIP_IGNORE_INSTALLED=0 pip install --user /builds/worker/checkouts/taskgraph &&
+                                            PIP_IGNORE_INSTALLED=0 pip install --user mozilla-version &&
```

Comment:
mozilla-version is used mainly to support github-releases. We may be able to remove it once we drop support.

Reply:
I'm unsure whether we could remove it. I'd say: let's not put a misleading TODO for now. I'll revisit once I know more.